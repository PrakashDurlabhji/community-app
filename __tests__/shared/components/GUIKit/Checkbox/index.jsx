import React from 'react';
import Renderer from 'react-test-renderer/shallow';

import Checkbox from 'components/GUIKit/Checkbox';


const rnd = new Renderer();

it('Default render', () => {
  rnd.render((<Checkbox />));
  expect(rnd.getRenderOutput()).toMatchSnapshot();
});
