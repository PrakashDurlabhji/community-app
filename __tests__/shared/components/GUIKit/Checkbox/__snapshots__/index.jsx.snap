// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default render 1`] = `
<label
  className="src-shared-components-GUIKit-Checkbox-___style__container___2a1GZ src-shared-components-GUIKit-Checkbox-___style__smSize___3aux-"
>
  <input
    checked={false}
    onChange={[Function]}
    type="checkbox"
  />
  <div
    className="src-shared-components-GUIKit-Checkbox-___style__checkmark___1xdsj"
  >
    <img
      alt="checkmark-icon"
      className="src-shared-components-GUIKit-Checkbox-___style__after___2rEBg"
      src="/community-app-assets/images/07507a04e0986d806cbd3ec73c724143.png"
    />
  </div>
</label>
`;
